BaseLib/ConfigTree.cpp

```diff
@@ -7,9 +7,12 @@
  *
  */
 
-#include <logog/include/logog.hpp>
 #include "ConfigTree.h"
 
+#include <logog/include/logog.hpp>
+
+#include "StringTools.h"
```

Comment:
I would like a separate `BaseLib/Logger.h` header, which includes logog and StringTools, better.